.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## BACKGROUND
+* Add information about the background and reason for the change.
+
+-----------------------------------------------------------------------
+
+## Changes
+* List the changes that you made
+
+-----------------------------------------------------------------------
+
+## Checklist
+- [ ] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
+------------------------------------------------------------------------
```

Comment:
Suggest to add a section for the README:
https://github.com/digitalocean/marketplace-kubernetes/blob/master/docs/LISTING-README-TEMPLATE.md

Reply:
A plus would be an integration test we can leverage.

Reply:
i'll add that to the updating section.